pkgs/by-name/ff/ffts/package.nix

```diff
@@ -0,0 +1,41 @@
+{
+  stdenv,
+  lib,
+  cmake,
+  git,
+  fetchFromGitHub,
```

Comment:
Please fix styling here, you may use `nixpkgs-fmt` to format the file.
You can also use https://github.com/jtojnar/nixpkgs-hammering as a linter.